marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubSecurityAdminTest.java

```diff
@@ -195,6 +195,22 @@ void task5CreateRoleInheritingPrivilegeThatDataHubSecurityAdminDoesntPossess() {
         }
     }
 
+    @Test
```

Comment:
This should go in a separate test - DataHubHttpUserTest, for consistency with the other tests.